tests/test_advanced_filters.py

```
@@ -0,0 +1,2120 @@
+from datetime import datetime, timedelta, timezone
+
+import pytest
+from fastapi.testclient import TestClient
+from nanoid import generate as generate_nanoid
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from src.models import Peer, Workspace
+
+
+@pytest.mark.asyncio
+async def test_logical_operators_and_filters(
```

Ruff (0.11.9)

Missing return type annotation for public function `test_logical_operators_and_filters`

Add return type annotation: `None`

(ANN201)